rulesengine/rules/utils/validators.py

```diff
@@ -12,11 +12,18 @@
     ('rewrite-headers', 'Rewrite playback headers'),
 )
 
+ENVIRONMENT_CHOICES = (
+    ('prod', 'Production'),
+    ('qa', 'QA/development'),
```

Comment:
I think we should probably name this e.g. `"test", "Testing/deveopment"` -- we have too many things named "QA" already.